notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>This is too many products to download. I try to avoid downloading anything larger than 50MB, if I can help it. 500+MB of files, especially if we're only using one, seems like overkill.</p><p> </p><p>You can use <code>extension='drz.fits'</code> to get the file you end up using, but we'd probably need to motivate only requesting the drizzled files.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>